server/src/dtos/user.dto.ts

```diff
@@ -31,15 +31,26 @@ export class UserResponseDto {
   profileImagePath!: string;
   @ApiProperty({ enumName: 'UserAvatarColor', enum: UserAvatarColor })
   avatarColor!: UserAvatarColor;
+  license!: UserLicense | null;
+}
+
+export class UserLicense {
+  licenseKey!: string;
+  activationKey!: string;
+  activatedAt!: Date;
 }
 
 export const mapUser = (entity: UserEntity): UserResponseDto => {
+  const license = entity.metadata?.find((item): item is UserMetadataEntity<UserMetadataKey.LICENSE> => {
+    return item.key === UserMetadataKey.LICENSE;
+  })?.value;
```

Comment:
I think this can just be `(item): item is UserMetadataEntity<UserMetadataKey.LICENSE> => item.key === UserMetadataKey.LICENSE`, right?